comp/rdnsquerier/impl/rdnsquerier_test.go

```diff
@@ -0,0 +1,222 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build test
```

Comment:
No need to dd the test build tag on files that end in `*_test.go` 😄

Reply:
Removed in commit 04687a25587bb09f3fe0a7e2fdfe8ead0279e98c.